File: app/src/main/java/com/shentu/wallpaper/model/entity/WallpaerSection.java
package com.shentu.wallpaper.model.entity;

import com.chad.library.adapter.base.entity.SectionEntity;

public class WallpaerSection extends SectionEntity<Wallpaper> {
    public WallpaerSection(boolean isHeader, String header) {
        super(isHeader, header);
    }

    public WallpaerSection(Wallpaper wallpaper) {
        super(wallpaper);
    }
}

File: app/src/main/java/com/shentu/paper/model/entity/WallpaerSection.kt
package com.shentu.paper.model.entity

import com.chad.library.adapter.base.entity.SectionEntity

class WallpaerSection : SectionEntity<Wallpaper?> {
    constructor(isHeader: Boolean, header: String?) : super(isHeader, header) {}
    constructor(wallpaper: Wallpaper?) : super(wallpaper) {}
}